apps/homescreen/test/unit/pages_test.js

```diff
@@ -397,5 +397,14 @@ suite('Pages', () => {
         assert.isFalse(editModeStub.called);
       });
     });
+
+    suite('resize', () => {
+      test('should call pages.synchronise()', () => {
+        var synchroniseCalled = false;
+        pages.pages.synchronise = () => { synchroniseCalled = true; };
```

Comment:
In this case, pages.pages is mocked with a div and the synchronise function isn't present - felt overkill to define the function, then use sinon when I could just make the function do what's needed for the test.